logspout/modules.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
It gets included with the ONBUILD commands:

https://github.com/gliderlabs/logspout/blob/c2bafe3ea23a2372cf4d72bfc06785d819ad35d7/Dockerfile#L9-L11